src/server/index.js

```diff
@@ -78,9 +77,20 @@ if (program.staticDir) {
     }
     logger.log(`=> Loading static files from: ${staticPath} .`);
     app.use(express.static(staticPath, { index: false }));
+
+    // If there's a favicon on the specified path, use it.
+    const faviconPath = path.resolve(staticPath, 'favicon.ico');
+    if (fs.existsSync(faviconPath)) {
+      app.use(favicon(faviconPath));
+    }
   });
 }
 
+// The first call to app.use(favicon) wins so we first check if there's a
```

Comment:
It's just something I noticed, but you're right that it's not super clear. I'll update this to use `hasCustomFavicon` based on your feedback.